mod-source-record-manager-server/src/main/java/org/folio/services/util/EventHandlingUtil.java

```diff
@@ -43,7 +43,7 @@ private EventHandlingUtil() {
     LOGGER.debug("Starting to send event to Kafka for eventType: {}", eventType);
     Event event;
     try {
-      event = createEvent(eventPayload, eventType, tenantId, true);
+      event = createEvent(eventPayload, eventType, tenantId, false);
```

Comment:
@bisecomsergiy  I think that method createEvent should be changed. The last argument should be deleted at all, and zipping inside should be removed as well.